hnn_core/cell_response.py

```diff
@@ -370,6 +370,13 @@ def plot_spikes_hist(self, trial_idx=None, ax=None, spike_types=None,
             Valid strings also include leading characters of spike types
 
             | Ex: ``'ev'`` is equivalent to ``['evdist', 'evprox']``
+        color : str | list of str | None
+            String input defining color of histograms plotted. If list of str
+            provided, histograms for each cell type will be plotted by cycling
+            through colors in the list.
+
+            | Ex: ``'r'``, ``['r', 'g']``
+        If None, default color cycle used.
```

Comment:
how do we know which color corresponds to which cell type?

Reply:
The legend will indicate the cell type in the plot, but in the code it can be redefined.

Reply:
I was thinking a dict might be more useful?

```py
dict(evprox='r', evdist='g')
```

Reply:
Defining a color sequence that gets cycled over is pretty standard for matplotlib and other high-level plotting libraries (e.g., seaborn) is it not? I'm not opposed to using a dictionary, but it's not as much of a lightweight solution as what is implemented now.

Something I just thought of, do we want it to also accept the RGBA color specification as well?

Reply:
since `net.gid_dict` is a dictionary, it means that the cell types won't always necessarily be in the same order before Python 3.6? Maybe I'm splitting at hairs here, but we should avoid sources of bugs

Reply:
I don't think that would be a bug. This function currently allows users to specify the cell types, drive types, or grouping of types they wish to plot. If the user makes no underlying assumptions about the order in which the various histograms will be plotted, they can rely on the default behavior which is to plot all cell types based on `net.gid_dict`. If we allow a color sequence to be entered, they could also theoretically use a matplotlib colormap!

Reply:
Would it be a bug that we don't guarantee plots having the same color if python versions change? Also that means that our current code also doesn't have that assurance, so we're not losing anything necessarily